README.md

```diff
@@ -250,16 +276,28 @@ user.subscribe(name: "default", plan: "default", quantity: 3)
 
 Name is an internally used name for the subscription.
 
-##### Plan
+###### Plan
 
 Plan is the plan ID or price ID from the payment processor. For example: `plan_xxxxx` or `price_xxxxx`
 
-##### Options
+###### Options
 
 By default, the trial specified on the subscription will be used.
 
 `trial_period_days: 30` can be set to override and a trial to the subscription. This works the same for Braintree and Stripe.
 
+##### Paddle
+It is currently not possible to create a subscription through the API. Instead the subscription in Pay is created by the Paddle Subscription Webhook. In order to be able to assign the subcription to the correct owner, the Paddle [passthrough parameter](https://developer.paddle.com/guides/how-tos/checkout/pass-parameters) has to be used for checkout.
+
+```javascript
+Paddle.Checkout.open({
+	product: 12345,
+	passthrough: "{\"owner_id\": 98765, \"owner_type\": \"User\"}"
+});
+```
```

Comment:
Is this secure? I could change the model and ID to anything.

Seems like we should do a signed GlobalID or something instead that we could verify server-side.

Reply:
Yes, you're right. That would allow to pay for another owner id or owner type. I changed it and now a signed GlobalID is used. In addition, a purpose string ensures that GlobalIDs cannot be exchanged between Paddle products.

Reply:
Since this is received by the webhook for the subscription, I think the `for: ` is probably unnecessary?

If so, we can remove it and simplify a little bit which is nice.

Reply:
It depends:  If you can create subscriptions for multiple owner types in your app or signed global IDs are made public otherwise, the `for:` would be necessary to prevent swapping the SGID in the javascript code.

```javascript
Paddle.Checkout.open({
	product: 12345,
	passthrough: "{\"owner_sgid\": \"<%= modelA.to_sgid.to_s %>\"}"
});
Paddle.Checkout.open({
	product: 67890,
	passthrough: "{\"owner_sgid\": \"<%= modelB.to_sgid.to_s %>\"}"
});
``` 
How likely is such a scenario? Probably not very likely. To be honest: If I pay for a subscription I am not interesting in changing the owner. That is the reason I just used `owner_type` and `owner_id` in the beginning. I don't know, maybe using SGIDs without `for:` is a good tradeoff between simplicity and security. What do you think?

Reply:
At least with an SGID, they could only swap SGIDs they had access to since it'd have to be generated server side. That'd leave it only to accounts they have access to.

If it was clear text, it's easy to tamper with. To be fair, you wouldn't want to pay for something and get it assigned to someone else, so it seems illogical to tamper with.

Maybe I'll build a helper to generate the passthrough JSON.